src/OrchardCore.Modules/OrchardCore.Features/Recipes/Executors/FeatureStep.cs

```diff
@@ -34,8 +34,8 @@ public Task ExecuteAsync(RecipeExecutionContext context)
             var step = context.Step.ToObject<FeatureStepModel>();
             var features = _extensionManager.GetFeatures();
 
-            var featuresToDisable = features.Where(x => step.Disable.Contains(x.Id)).ToList();
-            var featuresToEnable = features.Where(x => step.Enable.Contains(x.Id)).ToList();
+            var featuresToDisable = features.Where(x => step.Disable?.Contains(x.Id) == true).ToList();
```

Comment:
the `== true` is unnecessary

Reply:
Without == true the answer can be null which did not compile.

Reply:
Could do:
```
(x => step.Disable?.Contains(x.Id) ?? true)
```

Perhaps that is clearer?

Reply:
I think @NogginBox is correct, you need the `== true` because it's a `bool?`